uaclient/status.py

```diff
@@ -682,18 +684,28 @@ def _format_status_output(status: "Dict[str, Any]") -> "Dict[str, Any]":
 
     # We don't need the origin info in the json output
     status.pop("origin", "")
+
+    status["config_path"] = cfg["config_path"]
+
+    # We could instead delete the key from the cfg dict.
+    # We could copy the dict and delete the key, but this
+    # approach is simpler
+    status["config"] = {k: v for k, v in cfg.items() if k != "config_path"}
```

Comment:
I agree with the more "functional" style of creating a new dict based on the old one, rather than doing a full copy and mutating the copy. +1

> Random aside: sometimes dict/list comprehensions are hard to read, but this is one of those cases where it perfectly expresses a map, filter, and reduce all in one succinct line. Love it

My only thought to consider is that we're currently passing `cfg` to this function to do this here, but we construct most of the data in `config.py` where we already have `cfg` available. What are your thoughts on adding the config data here vs in `UAConfig._get_config_status()`?

Reply:
That's a good idea, I will refactor the code to use it